internal/iam/command_rbac_role_binding_list.go

```diff
@@ -108,7 +108,7 @@ func (c *roleBindingCommand) newListCommand() *cobra.Command {
 		pcmd.AddContextFlag(cmd, c.CLICommand)
 	}
 
-	cmd.Flags().String("resource", "", "Resource type and identifier using "<Prefix>:<ID>" format (for example, "User:u-123456"). If specified with "--role" and no principals, list all principals and role bindings.")
+	cmd.Flags().String("resource", "", "Resource type and identifier using \"<Prefix>:<ID>\" format (for example, \"User:u-123456\"). If specified with "--role" and no principals, list all principals and role bindings.")
```

Comment:
```suggestion
	cmd.Flags().String("resource", "", `Resource type and identifier using "<Prefix>:<ID>" format (for example, "User:u-123456"). If specified with "--role" and no principals, list all principals and role bindings.`)
```

Reply:
In the docs, we have a mix of realistic and simplistic IDs. I'd rather have more realistic ones. I'm creating a list of IDs to use for examples. Do you have a list of the IDs used for examples in the CLI?

Also, I noted the use of single quotes to avoid the backslashes.